RELEASING.adoc

```diff
@@ -22,3 +22,26 @@ For GA releases this should be configured under the `central` Maven profile.
 . Make sure that the release tag was automatically added by the release process.
 
 . Make sure that the link:https://github.com/spring-io/initializr/blob/master/initializr-service/src/main/resources/application.yml[Spring Boot Initializr] is updated to support the new version of Spring Cloud GCP.
+
+== Uploading Javadocs
+
+After releasing a new version of Spring Cloud GCP, you will need to upload the javadocs for the released version.
+By this point, we assume that a new release branch has been created for project (such as `1.1.x`, `1.2.x`, etc.).
+
+All javadocs are uploaded to the following URL:
+```
+https://googleapis.dev/java/spring-cloud-gcp/${LIBRARY_VERSION}/index.html
+```
+
+The latest version of the docs can be accessed at the `latest/` url:
+https://googleapis.dev/java/spring-cloud-gcp/latest/index.html
+
+1. Locate and open the `prod:cloud-java-frameworks/spring-cloud-gcp/publish_javadoc` Kokoro job.
+
+2. Add a new Github SCM and fill out the following parameters:
+
+    - SCM Name: Set this to `spring-cloud-gcp`
+    - Committish: Set this to the name of the new release branch or tag that you want to publish javadocs for (like `1.2.x` or `v1.2.1.RELEASE`).
+      Note that you must already have the `.kokoro/` scripts in the branch that you wish to publish javadocs for.
+
+3. Run the job.
```

Comment:
Add a verification step? What would be the URL?

Reply:
Done.